main.go

```diff
@@ -122,6 +125,10 @@ func (configs ConfigsModel) print() {
 }
 
 func (configs ConfigsModel) validate() error {
+	if err := input.ValidateWithOptions(configs.InstallDefaults, "Yes", "No"); err != nil {
+		return fmt.Errorf("issue with inpout InstallDefaults: %s", err)
```

Comment:
I know, it is not your fault. But could you please replace all `inpout` quickly with `input`? There is a typo in it 🙂